src/Curl.CommandLine.Parser/Extensions/SpanExtensions.cs

```diff
@@ -1,14 +1,14 @@
-using CurlToCSharp.Constants;
+using Curl.CommandLine.Parser.Constants;
 
-namespace CurlToCSharp.Extensions;
+namespace Curl.CommandLine.Parser.Extensions;
 
-public static class SpanExtensions
+internal static class SpanExtensions
 {
     public static Span<char> TrimCommandLine(this Span<char> input)
     {
-        return input
-            .TrimStart(new ReadOnlySpan<char>(new[] { Chars.Escape, Chars.Space }))
-            .Trim();
+        return input.ToString()
```

Comment:
Why do we need this change?

Reply:
This is because of the [Trim method](https://learn.microsoft.com/en-us/dotnet/api/system.memoryextensions.trim?view=net-6.0#system-memoryextensions-trim-1(system-span((-0))-system-readonlyspan((-0)))) which is not available in netstandard2.1

Reply:
let's drop .netstandard2.1 then

Reply:
Updated the code. Currently targeting netcoreapp3.1
Cannot use netcoreapp3.0 because `System.Runtime.CompilerServices.Unsafe.targets(4, 5): System.Runtime.CompilerServices.Unsafe doesn't support netcoreapp3.0. Consider updating your TargetFramework to netcoreapp3.1 or later.`